src/components/AccountContainer.js

```diff
@@ -46,16 +47,28 @@ class AccountContainer extends Component {
     }
   }
 
+   componentDidMount(){
+    fetch('https://boiling-brook-94902.herokuapp.com/transactions')
+      .then( res => res.json() )
+      .then( data => this.setState({ transactions: data}) )
+  }
+
   handleChange(event) {
-    // your code here
+    let initialInfo = this.state.transactions
+    let filiteredInfo = initialInfo.filter( (data) => (data.description.includes(event) || data.category.includes(event)))
+    this.setState({
+      searchTerm: event,
+      filteredTrans: filiteredInfo
+    })
   }
 
+
   render() {
 
     return (
       <div>
-        <Search searchTerm={this.state.searchTerm} handleChange={"...add code here..."} />
-        <TransactionsList transactions={this.state.transactions} searchTerm={this.state.searchTerm} />
+        <Search searchTerm={this.state.searchTerm} handleChange={this.handleChange.bind(this)} />
+        <TransactionsList transactions={this.state.transactions} filtered={this.state.filteredTrans} searchTerm={this.state.searchTerm} />
```

Comment:
see above - should be able to just give this `TransactionList` component one list of transactions. How long or short the list is can be decided by doing a filter here.